main.go

```diff
@@ -92,7 +91,11 @@ func (s3 *S3) Open(name string) (http.File, error) {
 }
 
 func getObject(ctx context.Context, s3 *S3, name string) (*minio.Object, error) {
-	names := [4]string{name, name + "/index.html", name + "/index.htm", "/404.html"}
+	names := []string{name, name + "/index.html", name + "/index.htm"}
+	if sapFile != "" {
+		names = append(names, sapFile)
```

Comment:
Is sapfile a full path or a relative path ?

Reply:
> Is sapfile a full path or a relative path ?

Full path, every URL should put always to the same file.

Reply:
> Is sapfile a full path or a relative path ?

I made a typo. It's not SAP but SPA.
It's Spa = Single Page Application

Reply:
can you fix the environment variable and also inside the code from `sap -> spa` ?